os-datavirt/added/standalone-openshift.xml

```diff
@@ -525,14 +525,14 @@
             <preparedplan-cache infinispan-container="teiid-cache"/>
             <transport name="embedded"/>
             <transport name="odata">
-                <authentication security-domain="teiid-security"/>
+                <authentication security-domain="##ODATA_SECURITY_DOMAIN##"/>
```

Comment:
I guess we could make it generic to cover any possible future transports, but there won't be additional ones for the current version of Teiid